lib/davinci_dtr_test_kit/profiles/communication_request_group.rb

```diff
@@ -0,0 +1,37 @@
+require_relative 'communication_request/communication_request_read'
+require_relative 'communication_request/communication_request_validation'
+
+module DaVinciDTRTestKit
+  class CommunicationRequestGroup < Inferno::TestGroup
+    title 'CRD CommunicationRequest Tests'
+    short_description 'Verify support for the server capabilities required by the CRD CommunicationRequest Profile'
+    description %(
+      # Background
+
+    The CRD CommunicationRequest sequence verifies that the system under test is
+    able to provide correct responses for CommunicationRequest queries. These queries
+    must contain resources conforming to the CRD CommunicationRequest Profile as
+    specified in the DaVinci Coverage Requirements Discovery (CRD) v2.0.1
+    Implementation Guide.
+
+    # Testing Methodology
+    ## Read
+    This test sequence will first perform the required read associated
+    with this resource. The server SHOULD be capable of returning a
```

Comment:
I think being a little bit more specific and tying these to the inputs would make it clearer to users. Something like

`First, Inferno will attempt to read each <type> resource id provided in the <input name> input`

Reply:
Addressed in [Clarify profile description wording](https://github.com/inferno-framework/davinci-dtr-test-kit/pull/35/commits/193f6aeba996e7818159b505eeae4033afb1aa59)